TAGS.md

```diff
@@ -0,0 +1,59 @@
+# Tags
+
+This document aims to provide reasoning why `config.json` contains the `"tags"` it contains.
+
+It uses as a basis the Javacript's [TAGS.md](https://github.com/exercism/javascript/blob/main/TAGS.md) because anything that Javascript does can be done with Typescript, being a transpiled language, and adds/modifies with the perks Typescript brings on top of it.
```

Comment:
```suggestion
It uses as a basis the JavaScript's [TAGS.md](https://github.com/exercism/javascript/blob/main/TAGS.md) because anything that JavaScript does can be done with Typescript, being a transpiled language, and adds/modifies with the perks TypeScript brings on top of it.
```